tests/test_static.py

```diff
@@ -101,6 +101,31 @@ def test_static_file_pathlib(app, static_file_directory, file_name):
     assert response.body == get_file_content(static_file_directory, file_name)
 
 
+@pytest.mark.parametrize(
+    "file_name",
+    [
+        "test.file",
+        "decode me.txt",
+        "python.png",
+        "symlink",
+        "hard_link",
+    ],
+)
+def test_static_file_pathlib_relative_path_traversal(
+    app, static_file_directory, file_name
+):
+    """Get the current working directory and check if it ends with "sanic" """
+    cwd = Path.cwd()
+    if not str(cwd).endswith("sanic"):
+        pytest.skip("Current working directory does not end with 'sanic'")
+
+    file_path = "./tests/static/../static/"
```

Comment:
Would it be possible to remove the dependency on cwd (and the directory structure) by using setting file_path to:
```
str(
        Path(static_file_directory) / ".." / Path(static_file_directory).name
    )
```

Reply:
Possibly. I will take a look at it again. It was being a PITA and I just wanted to get something in for the test to run.

Reply:
Maybe we can try python [tempfile module](https://docs.python.org/3/library/tempfile.html) for the testing files.